akka-typed/src/main/scala/akka/typed/persistence/scaladsl/PersistentActor.scala

```diff
@@ -41,19 +41,26 @@ object PersistentActor {
    * Factories for effects - how a persitent actor reacts on a command
    */
   object Effect {
-    def persist[Event, State](event: Event): Effect[Event, State] =
-      new Persist[Event, State](event)
 
-    def persistAll[Event, State](events: im.Seq[Event]): Effect[Event, State] =
+    def persist[Event, State](event: Event, events: Event*): Effect[Event, State] =
+      if (events.nonEmpty)
+        Effect.persist(im.Seq(event) ++ events)
+      else
+        new Persist[Event, State](event)
```

Comment:
```
if (events.nonEmpty) Effect.persist(event +: events)
else new Persist[Event, State](event)
```

?

Reply:
@ktoso see my comment below. I posted it before seeing this comment from you.